sru-scripts/manual/oracle-sru

```diff
@@ -47,21 +47,21 @@ ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- dpkg-query --show cloud-init
 ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- sudo hostname SRU-didnt-work
 ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- sudo rm -f $NETCFG_FILE
 ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- sudo cloud-init clean --logs --reboot || true
-ssh-keygen -f "/home/ubuntu/.ssh/known_hosts" -R $VM_IP
+ssh-keygen -f "${HOME}/.ssh/known_hosts" -R $VM_IP
 
 sleep 10  # Wait for the instance to actually go down
 while ! ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- cloud-init status --wait --long; do
     sleep 1
 done
 
 ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- hostname
-ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- ! grep Trace /var/log/cloud-init.log
+ssh "${SSHOPTS[@]}" ubuntu@$VM_IP -- ! grep Trace /var/log/cloud-init.log || true
```

Comment:
@TheRealFalcon Might as well add a the comment there for future us:
# Expecting OpenStack datasource to have two Traces one related to Oracle's lack of support of stock OpenStack datasource "meta-data" route and one related to EphemeralDCHP falling over because the ip route already exists (due to iscsi_root)